.gitlab-ci.yml

```diff
@@ -158,6 +158,11 @@ parent-vars:
 
 .build-ink-example-contracts:                  &build-ink-example-contracts
   - set -o pipefail
+  # Copy the local `ink-waterfall` example contracts into `INK_EXAMPLES_PATH`
+  # so that they are build and can be tested as well. They'll also show up
+  # in the regression bot statistics then.
+  - cp -r ./examples/ ${INK_EXAMPLES_PATH}
```

Comment:
Why do we need to copy it if we change `fn example_path` to look inside `./examples`?

Reply:
We copy them over to `INK_EXAMPLES_PATH` in the CI, since it's a minimal change which makes all following logic (storing contract sizes to redis, posting regressions to GitHub, etc.) work without having to change anything.

The fallback for looking in `./examples` if the contract is not found in `INK_EXAMPLES_PATH` is needed so that the waterfall can still be run locally (the copy happens only in the CI). I've added a clarifying comment.